lib/menu.rb

```diff
@@ -0,0 +1,28 @@
+class Menu
```

Comment:
I wonder if the logic for choosing a player mark should go in here...

Reply:
I thought about putting it there, but then I thought the GameFactory was in charge of setting everything up. I can see why the Menu would also be a viable option because they are making all the choices there. Anyone else have any thoughts?